zendesk/ticket.go

```diff
@@ -9,33 +9,64 @@ import (
 	"time"
 )
 
+type CustomField struct {
+	ID int64 `json:"id"`
+	// Valid types are string or []string.
+	Value interface{} `json:"value"`
+}
+
+// Custom Unmarshal function required because a custom field's value can be
+// a string or array of strings.
+func (cf *CustomField) UnmarshalJSON(data []byte) error {
+	var temp map[string]interface{}
+	if err := json.Unmarshal(data, &temp); err != nil {
+		return err
+	}
+
+	cf.ID = int64(temp["id"].(float64))
+
+	switch v := temp["value"].(type) {
+	case string:
```

Comment:
might want to add a default case that returns an error if this is an unexpected type.

Reply:
Added.